app/views/case-progress.html

```diff
@@ -0,0 +1,51 @@
+{% extends "includes/layout.html" %}
+
+{% block content %}
+
+<main id="content" role="main">
+
+{% include "includes/breadcrumbs.html" %}
+
+{% include "includes/validation-error-messages.html" %}
+
+<h2 class="heading-large">
+    <span class="heading-secondary">
+        {{ subTitle | title }}
+    </span>
+    {{ title | title }}
+</h2>
```

Comment:
A nice improvement here might be to extract this into a partial like the breadcrumbs, as we're now using it multiple places, and will likely need it on most pages to come.